REPOSITORY-GUIDELINES.md

```diff
@@ -1,101 +1,189 @@
+  - All open issues and PRs are closed
+  - All external collaborates are removed
+  - All webhooks, apps, integrations, or services are removed
+  - GitHub pages are disabled
+  - The repo is marked as archived using GitHub's archive feature
+  - The removal is announced on the knative-dev mailing list
+
+This procedure maintains the complete record of issues, PRs, and other
+contributions. It leaves the repository read-only and makes it clear that the
+repository is retired and no longer maintained.
+
+## Additional / Experimental Repositories
+
+We want Knative working groups to be able to own code outside the core `knative`
+organization, which is kept intentionally small and has a high bar for entry.
+The `knative-sandbox` github organization was created for this purpose.
+Repositories in `knative-sandbox` are intended to give working groups more
+latitude to experiment with new ideas and to self-organize and manage
+contributions which may be important to the project but which do not need the
+level of governance of the core project.
+
+Note that `knative-sandbox` is **not** an incubation area for projects entering
+the core; in some cases, projects in the sandbox will remain there for the
+entire duration. Promotion from the sandbox into core will be handled on a
+case-by-case basis by joint decision of the Steering Committee and the Technical
+Oversight Committee.
+
+### Mechanics of working-group owned repositories in `knative-sandbox`
+
+#### Creation
+
+Note: prior art here from the
+[Kubernetes community](https://github.com/kubernetes/community/blob/master/github-management/kubernetes-repositories.md).
+
+Working groups should be able to create repositories that meet the following
+bar:
+
+- Vendor independent (no direct links to specific clouds, no required
+  dependencies on vendor-specific tools)
+
+- Homed within an existing working group (if you want a new working group too,
+  create the working group first)
+
+- Must adopt the Knative CLA
+
+- Must adopt the Knative
+  [code of conduct](https://github.com/knative/community/blob/master/CODE-OF-CONDUCT.md)
+
+- Must adopt the Apache 2.0 license
+
+- Must be sponsored by a specific working group, which should be designated in
+  the `README.md` for the repo. In addition, at least one WG lead must be in the
+  root-level ONWERS file.
+
+- Copyright assigned to The Knative Authors
+
+- Document clear release and install processes.
+
+- APIs must be homed in a valid group with a DNS prefix approved by the Steering
+  Committee.
+
+  - APIs under `knative.dev` should:
+
+    1. Be under a subdomain managed by the owning Working Group (e.g.
+       `sources.knative.dev`, not `knative.dev`). The TOC is considered the
+       authority for assigning subdomains to working groups.
+
+    1. Coordinate with the Working Group in API naming to avoid collisions or
+       confusing naming. This may also include API review to ensure consistency
```

Comment:
I think we should also set the general expectation of API review for alignment with accepted norms / conventions (but recognize this is a big can of worms to open) - maybe let's do this in a follow-up.